wagtail_content_import/mappers/converters.py

```diff
@@ -138,7 +138,12 @@ def import_as_image_model(name, content, owner, title=None):
 
         # Set image file hash
         image.file.seek(0)
-        image._set_file_hash(image.file.read())
+        try:
+            image._set_file_hash(image.file.read())
+        except TypeError:
+            # This argumentless version was introduced in Wagtail 4.2.2
```

Comment:
Relevant commit if you want to link it in the comment:

https://github.com/wagtail/wagtail/commit/3c0c64642b9e5b8d28b111263c7f4bddad6c3880